lib/course_planner/users.ex

```diff
@@ -33,4 +32,9 @@ defmodule CoursePlanner.Users do
       error -> error
     end
   end
+
+  def notify_user(%{id: id}, %{id: id}, _), do: nil
+  def notify_user(modified_user, _, notification_type) do
```

Comment:
could you rename `_` to `_modifier`, `_current_user` or something which makes sense but with a name?